sagemaker-training-compiler/huggingface/tensorflow_multiple_gpu_single_node/language-modeling.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # SageMaker Training Compiler has increased the memory consumption of the model leading to decrease in batch size.
</pre><p>Above, you say: <code> this manifests as a reduction in memory utilization and a consequent increase in the largest batch size that can fit on the GPU</code> - but here you are saying training compiler increases memory consumption and decreases batch size - be consistent in explaining the effects of using this feature.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Yes, this only applies to FP16. I have removed this line since this is a FP32 notebook